### ISSUE TITLE
{% Include %} - is it possible to create variables on the same line

### ISSUE THREAD

**Issue by tatarincev** [lunet-io/scriban#106]
Is it by design what this line doesn't work? And if yes, can you add support for this syntax?
```
{% include 'snippet', my_variable: 'apples', my_other_variable: 'oranges' %}
```

**Comment by xoofx** [comment]
Yeah, I don't remember exactly but I think I discovered that this was not well standardized in liquid, so I couldn't find a common syntax for it. It is also annoying to make it different from the scriban `include`. But I frankly don't remember the details.

I'm going to be off on features for a while concerning scriban (shifting my sparetime to other projects), so if you want a support for this, PR welcome.

**Comment by tatarincev** [comment]
@xoofx  I'll do. Thanks you for great job.

**Comment by skironDotNet** [comment]
@tatarincev  I don't think what you are asking makes sense, at least not in context of the scriban. All your child templates loaded by {{ include 'child.sbn' }} use variables from the global variable list, and the global variable list support objects so in your child template you can have something like `{{ order.number }} and {{ user.name }}`. Passing variables in include makes no sense in context of templating, this not Angular app with components, the template is "dumb" and all it's looking for is the variables in the variable list. The concept of include is just to be able to re-use some templates but the variable injection still happens from global point of view, the global variable list just must provide the variables.
